packages/tabs/src/LionTabs.js

```diff
@@ -267,11 +268,11 @@ export class LionTabs extends LitElement {
       return newIndex;
     }
     if (direction === 'ArrowRight' || direction === 'ArrowDown') {
-      const nextNotDisabledTab = this.__getNextNotDisabledTab(tabs, currentTab);
+      const nextNotDisabledTab = this.__getNextNotDisabledTab(tabs, currentTab, 'right');
       return tabs.findIndex(tab => nextNotDisabledTab === tab);
     }
     if (direction === 'ArrowLeft' || direction === 'ArrowUp') {
-      const nextNotDisabledTab = this.__getNextNotDisabledTab([...tabs].reverse(), currentTab);
```

Comment:
This trick doesn't work. If you have let's say 7 tabs, and you're on number 5, pressing arrow left would reverse that to be number 2 and focus would shift to 1. Fixed in my commit